compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -399,9 +414,29 @@ impl<'context> Elaborator<'context> {
         }
 
         self.run_attributes_on_functions(functions, &mut generated_items);
+
         generated_items
     }
 
+    fn run_attributes_on_modules(
+        &mut self,
+        module_attributes: &Vec<ModuleAttribute>,
```

Comment:
```suggestion
        module_attributes: &[ModuleAttribute],
```
Surprised clippy doesn't give a lint here